src/transform.js

```diff
@@ -326,7 +326,14 @@ const transform = {
       }
       const typeAnnotation = t.tsTypeAnnotation(returnType);
       path.replaceWith(
-        t.tsFunctionType(typeParameters, parameters, typeAnnotation)
+        !path.parent ||
```

Comment:
When would `path` not have a parent?

Reply:
I don't know, I guess `path.parent` can be `undefined`.